packages/core/botpress/src/botpress.js

```diff
@@ -68,9 +69,7 @@ import packageJson from '../package.json'
 
 import createServer from './server'
 
-import { getDataLocation, getBotpressVersion } from './util'
-
-import { isDeveloping, print } from './util'
+import { getDataLocation, getBotpressVersion, validateVersion, isDeveloping, print } from './util'
```

Comment:
getBotpressVersion, isDeveloping, print are imported but unused

Reply:
used [getBotpressVersion](https://github.com/botpress/botpress/blob/master/packages/core/botpress/src/botpress.js#L134) and [isDeveloping](https://github.com/botpress/botpress/blob/master/packages/core/botpress/src/botpress.js#L424)

Reply:
Oh, I'm sorry I didn't notice there were _two_ imports from the same file here which you just joined.